### PR TITLE
Add SyncStatus icons to header, rows

### PR DESCRIPTION
**Pull request by jakob223**
Issue: KBFS-4021

### CODE REVIEW

**Comment by jzila** [comment]
@jakob223 description still says WIP. Is this actually ready? Also you might want to make this a merge into @songgao's branch for now, so the diff doesn't also capture his changes.

**Comment by jakob223** [comment]
@jzila it is ready, whoops. I did mark it as merging into song's branch.

**Comment by jakob223** [comment]
@jzila it is ready, whoops. I did mark it as merging into song's branch.

File: shared/common-adapters/list-item2.js [hunk 1]
```diff
@@ -85,7 +101,7 @@ export const smallHeight = Styles.isMobile ? 56 : 48
 export const largeHeight = Styles.isMobile ? 64 : 56
 const smallIconWidth = Styles.isMobile ? 56 : 56
 const largeIconWidth = Styles.isMobile ? 72 : 72
-
+const statusIconWidth = Styles.isMobile ? 32 : 32
```
[the remark this answers is not in the document]
Comment (jakob223) [reply shared/common-adapters/list-item2.js]: As far as I can tell from the designs, the icons are different sizes but the box containing them is the same size.

**Pull request merged**